scalafix-core/shared/src/main/scala/scalafix/internal/rule/Disable.scala

```diff
@@ -1,18 +1,59 @@
 package scalafix.internal.rule
 
 import metaconfig.{Conf, Configured}
+
 import scala.meta._
+import scala.meta.transversers.Traverser
+import scalafix.config.CustomMessage
 import scalafix.internal.config.DisableConfig
-import scalafix.lint.LintCategory
-import scalafix.lint.LintMessage
-import scalafix.rule.SemanticRule
-import scalafix.rule.{Rule, RuleCtx}
-import scalafix.util.SemanticdbIndex
-import scalafix.util.SymbolMatcher
+import scalafix.internal.util.SymbolOps
+import scalafix.lint.{LintCategory, LintMessage}
+import scalafix.rule.{Rule, RuleCtx, SemanticRule}
+import scalafix.util.{SemanticdbIndex, SymbolMatcher}
+
+object Disable {
+
+  /**
+    * Searches for specific nodes in Tree and adds result of searching to the list.
```

Comment:
> A tree traverser to collect values with a custom context.
> 
> At every tree node, either builds a new context `U` or returns a new value `T` to accumulate.
> To collect all accumulated values, use `result(Tree)`.

Reply:
:D I'm not very experienced in writing docs, unfortunately.